userinterface/map/routeplannermodule.cpp

```diff
@@ -356,3 +356,13 @@ void RoutePlannerModule::drawRoute(QPainter& painter, QTransform drawTrans, QTra
         }
     }
 }
+
+void RoutePlannerModule::reverseRoute()
```

Comment:
To follow the pattern of the other member functions, this should be called reverseCurrentRoute(). reverseRoute would expect an index argument.